python/docs/source/getting_started/installation.rst

```diff
@@ -75,7 +75,7 @@ Note that `PySpark at Conda <https://anaconda.org/conda-forge/pyspark>`_ is avai
 Official Release Channel
 ------------------------
 
-Different flavors of PySpark is available in the `official release channel <https://spark.apache.org/downloads.html>`_.
+Different flavors of PySpark is available in the `official release channel <https://spark.apache.org/downloads.html>`__.
```

Comment:
Does this work? the link below depends on this anchor.
(We can fix is -> are too)

Reply:
See https://github.com/apache/spark/pull/29735 for an alternative

Reply:
I only compared the result html file and the diff is (rhs is old):
```
435a436,438
> <div class="system-message">
> <p class="system-message-title">System Message: WARNING/2 (<tt class="docutils">python/docs/source/getting_started/installation.rst</tt>, line 20); <em><a href="#id3">backlink</a></em></p>
> Duplicate explicit target name: &quot;official release channel&quot;.</div>
```

which seems OK. 

Sure I'll abandon this PR since you have better knowledge on this :)